#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("current_job", int_t, 0, "current job", 1, 0, 16)
gen.add("slice_size", int_t, 0, "size of slice bar", 5, 2, 100)
gen.add("min_ray", int_t, 0, "min size as obstacle. Must smaller than slice_size", 3, 1, 100)
gen.add("footprint", str_t, 0, "Cac dinh cua field", '')
gen.add("publish_vel", bool_t, 0, "", True)
gen.add("cmd_vel_input", str_t, 0, "topic cmd vel input", '/cmd_vel')
gen.add("cmd_vel_output", str_t, 0, "topic cmd vel output", '/safety_cmd_vel')
exit(gen.generate("scan_safety", "scan_safety", "Safety"))
